Server/bank/database/src/main/java/com/cashmanager/server/database/entities/Account.java

```diff
@@ -0,0 +1,49 @@
+package com.cashmanager.server.database.entities;
+
+import com.cashmanager.server.database.enums.AccountState;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Entity(name = "accounts")
+public class Account {
+    @Id
+    private UUID id;
```

Comment:
You can choose a auto-generate strategy for the primary key in the database
```suggestion
@GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;
```